src/features/moreTab/more/MoreScreen.tsx

```diff
@@ -295,7 +295,7 @@ const MoreScreen = () => {
   )
 
   return (
-    <SafeAreaBox backgroundColor="primaryBackground">
+    <SafeAreaBox backgroundColor="primaryBackground" flex={1}>
```

Comment:
I noticed this screen sometimes doesn't display the entire list and gets cut off half way, maybe its related to this change? Not sure if this is a new or existing bug or why you had to change the flex here.

Reply:
I changed the tabs so they load lazily. The flex was a fix needed because of that. There must still be a little bug here though. Will fix.

Reply:
Setting `initialNumToRender={100}` seems to fix the bug. I also cleaned up the ts warning here.